public/locales_uid2/en.json

```diff
@@ -17,25 +17,26 @@
   "Powered by": "Powered by",
   "Internal Server Error": "Internal Server Error",
   "EXIT": "EXIT",
-  "Welcome!": "Welcome!",
-  "trusted-partners-committed": "Our trusted advertising partners using the UID identifier are committed to delivering relevant advertising that also respects your choice.",
-  "Advertising-enables-access": "Advertising enables access to your favorite content across the internet.",
+  "Welcome!": "Hello,",
   "Opt-Out": "Opt-Out",
-  "Your-choice-to-opt-out": "Your choice to Opt-Out will indicate to trusted advertising partners that you no longer wish to receive targeted advertising based on your UID.",
-  "Scope-is-limited": "The scope of this Opt-Out is limited to sites which use UID.",
+  "Your-choice-to-opt-out": "Your choice to opt out will indicate to trusted advertising partners that you no longer want to receive targeted advertising based on your UID2 identifier.",
+  "Scope-is-limited": "The scope of this opt-out is limited to sites that use Unified ID 2.0.",
   "For-choice-with-respect": "For choice with respect to advertising based on cookies, please visit ",
   "(English)": "",
-  "Note-choice-may-limit": "Note: this choice may limit your ability to access ad supported content.",
+  "Note-choice-may-limit": "Note: This choice might affect your ability to access ad-supported content.",
   "OPT OUT": "OPT OUT",
   "Are you sure?": "Are you sure?",
   "YES": "YES",
   "NO": "NO",
   "This choice may limit your ability to access ad supported content.": "This choice may limit your ability to access ad supported content.",
   "You-may-be-asked": "You may be asked to Opt-In again in the future to access such content.",
-  "We-have-received-your-request": "We have received your request to opt out and will begin processing it shortly.",
+  "We-have-received-your-request": "We have received your request to opt out of personalized advertising with UID2 identifier. Please allow 24 hours to complete your opt-out request.",
```

Comment:
Per my question in Figma (https://www.figma.com/file/3G7B4QXvJTQKSEfUSaCsss/TransparentAdvertising.org?type=design&node-id=614-225&mode=design&t=WLgWa9a12Rq8r1BK-0), I'm not sure if the timeframe of 24 hours is accurate.